### PR TITLE
Update license-of-site.md

### PR DESCRIPTION
**Pull request by samgnaniah**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: website/two-column-pages/docs/license-of-site.md [hunk 1]
```diff
@@ -35,5 +35,11 @@ Again, please link back to the original source page so that readers can refer to
 ### Other Media
 If you produce non-hypertext works, such as books, audio, or video, we ask that you make a best effort to include a spoken or written attribution in the spirit of the messages above.
 
+### Ballerina Website Attributions
+In the creation of this website, we used the following Creative Commons Attribution license: 
+
+* [Ballerina by Example](https://ballerina.io/learn/by-example/) by [mmcgrana](https://github.com/mmcgrana/gobyexample/) is licensed under [CC BY 2.0](https://creativecommons.org/licenses/by/2.0/)
```
Comment (samisawso2) [inline website/two-column-pages/docs/license-of-site.md]: It is not BBE by mmcgrana. 

Should be something like 
Ballerina by Example sample pages generated using a tooldeveloped by @mmcgrana, licensed under CC BY / modified from original

Comment (samgnaniah) [reply website/two-column-pages/docs/license-of-site.md]: Updated.

File: website/two-column-pages/docs/license-of-site.md [hunk 2]
```diff
@@ -38,8 +38,7 @@ If you produce non-hypertext works, such as books, audio, or video, we ask that
 ### Ballerina Website Attributions
 In the creation of this website, we used the following Creative Commons Attribution license: 
 
-* [Ballerina by Example](https://ballerina.io/learn/by-example/) by [mmcgrana](https://github.com/mmcgrana/gobyexample/) is licensed under [CC BY 2.0](https://creativecommons.org/licenses/by/2.0/)
-
+* [Ballerina by Example](https://ballerina.io/learn/by-example/) sample pages generated using a tool developed by [mmcgrana](https://github.com/mmcgrana/gobyexample/), licensed under [CC BY 2.0](https://creativecommons.org/licenses/by/2.0/) and modified from original.
```
Comment (samisawso2) [inline website/two-column-pages/docs/license-of-site.md]: The word "tool" has to be linked to https://github.com/mmcgrana/gobyexample/

and "@mmcgrana" to https://twitter.com/mmcgrana

Comment (samisawso2) [reply website/two-column-pages/docs/license-of-site.md]: Actually, let link "mmcgrana" to  https://github.com/mmcgrana

That is his git account.

**Comment by samgnaniah** [comment]
Updated.

**Pull request merged**